nipio/backend.conf

```diff
@@ -23,6 +23,14 @@ id=1
 hostmaster=hostmaster@nip.io.example
 # Name server
 ns=ns1.nip.io.example
+# Refresh
```

Comment:
Maybe specify that the unit is seconds.
"# Refresh (in seconds, 3600 is 60 minutes)"